client/maps/map-image.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import ImageIcon from '../icons/material/baseline-image-24px.svg'
+
+import { grey800 } from '../styles/colors'
+import { Subheading } from '../styles/typography'
+
+const ImgElement = styled.img`
+  display: block;
+  width: 100%;
+  object-fit: cover;
+`
+
+const NoImage = styled.div`
+  display: flex;
```

Comment:
These styles (especially the sizes) feel *very* particular to the map browser, and I'm not sure they should really be in here